docs/source/markdown/podman-container-clone.1.md

```diff
@@ -0,0 +1,175 @@
+% podman-container-clone(1)
+
+## NAME
+podman\-container\-clone - Creates a copy of an existing container
+
+## SYNOPSIS
+**podman container clone** [*options*] *container* *name* *image*
+
+## DESCRIPTION
+**podman container clone** creates a copy of a container, recreating all namespaces using the original options when the container was created allowing users to manipulate the original configuration. The second argument in this command can change the image of the clone container
```

Comment:
Maybe "recreating the original with an identical configuration" instead?

Also should mention the image bit